src/lambdas/archive-user-logs/src/index.ts

```diff
@@ -0,0 +1,30 @@
+import { gunzip } from "zlib"
+import { promisify } from "util"
+import type { Event } from "./types"
+import { ValidEvent, ValidInput } from "./types"
+import { uploadToS3 } from "./utils/uploadToS3"
+
+export const handler = async (input: unknown): Promise<void> => {
```

Comment:
Wonder if we can use a type here instead of `unknown`. Is that possible or the type is actually unknown?

Reply:
The type into a lambda is always by it's definition unknown. If we set a type here it's actually misleading because the type coming into the lambda could be different.